src/BaseClient.php

```diff
@@ -108,53 +136,137 @@ public function getToken(): ?array
      * @throws Exception when something unexpected went wrong.
      */
     public function authenticate(string $clientId, string $clientSecret): void
+    {
+        $tokenRequest = TokenRequest::constructFromArray([
+            'grant_type' => 'client_credentials',
+        ]);
+
+        $this->requestToken($clientId, $clientSecret, $tokenRequest, 'RETAILER');
```

Comment:
Klopt het dat je hier de `JWTToken` niet terug geeft?

Reply:
Dat klopt. De andere functies geven een refresh token terug. Bij deze authenticatie is er geen refresh token.